packages/types/src/api/web/global/email.ts

```diff
@@ -10,19 +11,18 @@ export enum EmailTemplatePurpose {
 }
 
 export interface SendEmailRequest {
-  workspaceId?: string
   email: string
-  userId: string
+  userId?: string
   purpose: EmailTemplatePurpose
   contents?: string
   from?: string
   subject: string
-  cc?: boolean
-  bcc?: boolean
+  cc?: string
+  bcc?: string
```

Comment:
These were just flatly wrong, assuming an oversight in the original TS conversion.